grand/sim/efield2voltage.py

```diff
@@ -47,7 +49,9 @@ class Efield2Voltage:
       * Save output in ROOT format
     """
 
-    def __init__(self, f_input, f_output="", seed=None, padding_factor=1.0):
+    def __init__(self, f_input, f_output="", seed=None, padding_factor=1.0, du_type='GP300'):
```

Comment:
The default model is the same that used in the master branch (Light_GP300Antenna_EWarm_leff.npz, Light_GP300Antenna_SNarm_leff.npz, Light_GP300Antenna_Zarm_leff.npz).

Reply:
I see. I'm thinking from the perspective of someone using the class. It would be good that the class documentation states what it is doing, and if the recomendation is to usea differrent parameter than the default, then state it here.